### DISCUSSION TITLE
how to hide/unhide the sidebar

### DISCUSSION THREAD

**Discussion by guoyejun** [inducer/pudb#595] [Q&A]
Hi, 

pudb is a very good debug tool, thanks!

Sometimes I want to share the copy during the debug, in other words, copy the code with several lines, but the sidebar is always included (even I press - for many times, but still a bit of width is there), possible to hide/unhide the sidebar? thanks.

**Comment by inducer** [comment]
That's not possible at the moment. I don't know whether Urwid allows zero-width columns. If so, that would likely be the most straightforward way. Just "not having" a sidebar is hard, because many bits of code in the debugger assume that the sidebar is there (e.g. by updating widget contents).

**Reply by guoyejun** [reply]
got it, thanks.
